### PR TITLE
Add Element X

### LINKED ISSUE #2060: Add Element X to clients

### PR DESCRIPTION
**Pull request by thibaultamartin**
Closes #2060

### CODE REVIEW

File: content/ecosystem/clients/element-x.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
++++
+title = "Element X"
+template = "ecosystem/client.html"
+[extra]
+thumbnail = "element-x.svg"
+maintainer = "Element"
+maturity = "Beta"
+repo = "https://github.com/vector-im"
+matrix_room = "#element-x-ios:matrix.org"
```
Comment (MTRNord) [inline content/ecosystem/clients/element-x.md:9]: Feels like an oversight of the data model that we only support one room when there are 2 rooms 😬 (iirc there is also an android room for element x specifically)

File: content/ecosystem/clients/element-x.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+threads = false
+sso = false
+multi_account = false
+multi_language = true
+[extra.packages]
+google_play_store.app_id = "io.element.android.x"
+apple_app_store = { app_id = "id1631335820", org = "vector" }
++++
+
+Element X is prioneering the Matrix 2.0 implementation, notably supporting fast sync with sliding sync.
```
Comment (MTRNord) [inline content/ecosystem/clients/element-x.md:28]:
```suggestion
Element X is pironeering the Matrix 2.0 implementation, notably supporting fast sync with sliding sync.
```

**Review by MTRNord — APPROVED** [review]
One typo. Otherwise lgtm

File: content/ecosystem/clients/element-x.md (hunk 1 above)
Comment (thibaultamartin) [reply content/ecosystem/clients/element-x.md:9]: Hm yes it's worth fixing it. We should definitely support the room variants. I wonder if we should change the data model more in-depth too.

I quite like having one tile per "project", even if it spans across several platforms

File: content/ecosystem/clients/element-x.md (hunk 2 above)
Comment (thibaultamartin) [reply content/ecosystem/clients/element-x.md:28]: pioneering maybe :D ?

Comment (MTRNord) [reply content/ecosystem/clients/element-x.md:28]: Fixed my suggestion :)

File: content/ecosystem/clients/element-x.md (hunk 1 above)
Comment (MTRNord) [reply content/ecosystem/clients/element-x.md:9]: I guess this kinda is also related to https://github.com/matrix-org/matrix.org/issues/2010

**Comment by thibaultamartin** [comment]
Let's add this project and rework the data model after

**Pull request merged**